### PR TITLE
l10n: add zh_{CN,TW} translation, 100%

### PR DESCRIPTION
**Pull request by MingcongBai**
Here is a zh_CN translation for Terminix, and finished.

### CODE REVIEW

File: po/zh_CN.po [hunk 1]
```diff
@@ -0,0 +1,721 @@
...
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Generator: Poedit 1.8.6\n"
+"Last-Translator: Jeff Bai <jeffbai@aosc.xyz>\n"
+"Plural-Forms: nplurals=1; plural=0;\n"
+"Language: zh_CN\n"
+
+# Terminix gettext pot file
```
Comment (Arthur2e5) [inline po/zh_CN.po]: Uh……What happened here?

Comment (MingcongBai) [inline po/zh_CN.po]: I believe it's just the pot's fault... I did not touch that part.

Comment (MingcongBai) [inline po/zh_CN.po]: And apparently all other po's have the same issue.

**Comment by MingcongBai** [comment]
@Arthur2e5 Thanks mate.

**Comment by MingcongBai** [comment]
zh_TW translation added, thanks to @Arthur2e5 .

**Comment by MingcongBai** [comment]
The reason why I am committing for @Arthur2e5 is that he's currently on a school trip to Italy... (well he's just too lazy to use git)

**Comment by frnogueira** [comment]
:+1:

File: po/zh_CN.po (hunk 1 above)
Comment (phw) [inline po/zh_CN.po]: That first entry with `msgid ""` and the following multiline `msgstr` is just the po header, nothing unusual here :)

Setting the project id and language team as you did is fine, but the file will work without it.

**Comment by MingcongBai** [comment]
screenshot_20160222_204123

Yay!

**Pull request merged**